app/lib/domain/controllers/tts_controller.dart

```diff
@@ -0,0 +1,17 @@
+import 'dart:io';
+import 'package:flutter/foundation.dart';
+import 'package:flutter_tts/flutter_tts.dart';
+import 'package:logger/logger.dart';
+import 'package:word_link/infrastructure/core/logger.dart';
```

Comment:
Everything here except of flutter_tts is no longer in use. Your IDE should tell you that.
Please delete